conbench/static/app.css

```diff
@@ -21,7 +21,6 @@ table.dataTable thead .sorting_desc {
 }
 
 table.dataTable td {
-  padding: 6px !important;
```

Comment:
With the additional column, depending on screen size, sometimes the `Name` column now gets cut off and wrapped to the "responsive" plus sign icon on the left side of the DataTable. I noticed that the plus sign gets uglily overlapped by the timestamp unless we get rid of this padding.